.storybook/styles/stories/variables.less

```diff
@@ -0,0 +1,49 @@
+.color-text {
+  color: @color-text
+}
+
+.color-text-light {
+  color: @color-text-light
+}
+
+.color-text-lighter {
+  color: @color-text-lighter
+}
```

Comment:
we already had classes for this ;) `text-color-default`, `text-color-default-light`,  `text-color-default-lighter`